widget/label.go

```diff
@@ -162,3 +177,36 @@ func (l *Label) updateFromData(data binding.DataItem) {
 	}
 	l.SetText(val)
 }
+
+type labelRenderer struct {
+	l *Label
+}
+
+func (r *labelRenderer) Destroy() {
+}
+
+func (r *labelRenderer) Layout(s fyne.Size) {
+	r.l.selection.Resize(s)
+	r.l.provider.Resize(s)
+}
+
+func (r *labelRenderer) MinSize() fyne.Size {
+	return r.l.provider.MinSize()
+}
+
+func (r *labelRenderer) Objects() []fyne.CanvasObject {
+	if !r.l.Selectable {
```

Comment:
nice optimization for the non-selectable case :)

Reply:
This allows us to keep the same XML for old usages of label too!